susemanager-sync-data/additional_products.json

```diff
@@ -10,7 +10,7 @@
         "friendly_name" : "AlmaLinux 8 x86_64",
         "product_class" : "SLE-M-T",
         "cpe" : null,
-        "free" : false,
+        "free" : true,
```

Comment:
since this file is generated we need to also adjust the data in the fake products in suma toolbox.

Reply:
I think I did it (MR) but I need to double check.

Reply:
Seems this is missing. I will do a MR

Reply:
I had the changes already on my disk. Now is the merge request created `merge_requests/281`